Makefile

```diff
@@ -28,7 +28,7 @@ dev-image:
 	docker inspect $(DEV_DOCKER_IMAGE) &> /dev/null || docker build --rm -t $(DEV_DOCKER_IMAGE) .
 
 ifeq ($(DOCKER), yes)
-TEST_PREFIX := docker run --rm $(DEV_DOCKER_IMAGE)
+TEST_PREFIX := docker run --rm --privileged -v /run:/run -v /tmp:/tmp -v `pwd`:/go/src/github.com/mesosphere/csilvm -v /dev:/dev --ipc=host $(DEV_DOCKER_IMAGE)
```

Comment:
Why does the docker container need to share /var, /run, and the host's IPC? Shouldn't LVM be completely self-contained in the docker container w/o needing ANYTHING from the host?

Not sure that the UCR container running the plugin shares the same things with the plugin...

@jieyu thoughts?

Reply:
I don't know about `/var` but `/run` appears to be necessary (removing it causes certain tests to fail). 

LVM relies on IPC, presumably for interprocess locking. Removing `--ipc=host` hangs the tests.

The same goes for `/tmp`. I think that has to do with the fact that the temporary files we create in the tests and use to back the loop devices need to be visible to the host.

Reply:
> Shouldn't LVM be completely self-contained in the docker container w/o needing ANYTHING from the host?
Please ignore this last question. Clearly I wasn't fully awake.

I *am* concerned that if we need additional mounts (like /run) for reliable operation, as well as host IPC, that we're properly configuring the plugin when building the Mesos protos.